--- splix/src/rastertospl2.cpp	2006/08/26 03:31:31	1
+++ splix/src/rastertospl2.cpp	2006/09/01 14:20:19	7
@@ -48,6 +48,14 @@
 		return 1;
 	}
 
+	// Create the document
+	document = new Raster(argv[1], argv[2], argv[3], argv[4], 
+		argv[5], argv[6]);
+	if (document->load()) {
+		delete document;
+		return 1;
+	}
+
 	// Open the PPD file
 	ppd = ppdOpenFile(getenv("PPD"));
 	ppdMarkDefaults(ppd);
@@ -61,14 +69,6 @@
 	// Create the printer
 	printer = new Printer(ppd);
 
-	// Create the document
-	document = new Raster(argv[1], argv[2], argv[3], argv[4], 
-		argv[5], argv[6]);
-	if (document->load()) {
-		delete document;
-		return 1;
-	}
-
 	// Convert and print
 	DEBUG("Génération du code....");
 	spl2.setPrinter(printer);
